pkg/tests/filesystem/file_system_test.go

```diff
@@ -189,3 +189,11 @@ func (fs *testFileSystem) Unmount(mountpoint string) error {
 
 	return nil
 }
+
+func (f *testFileSystem) GenerateRecoveryData() map[string]string {
+	return map[string]string{}
+}
+
+func (f *testFileSystem) LoadRecoveryData(map[string]string) {
+
+}
```

Comment:
Are these used by the test?

Reply:
No but it must conform to the File System API interface